package.json

```diff
@@ -8,7 +8,9 @@
     "test": "test"
   },
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "gulp test",
+    "preinstall": "npm install -g typings",
```

Comment:
We should be able to depend on a local `typings` package. IIRC as of npm 2 it will resolve local packages in other npm scripts, so `typings install` will still work if you install `typings` locally.